my-react-app/src/pages/Blogs.jsx

```diff
@@ -0,0 +1,7 @@
+const Blogs = () => {
+    return (
+      <h1>This is the Blog Page</h1>
+    );
+  };
+  
+  export default Blogs;
```

Comment:
This little red circle means you dont have a new line at the end of the file its good convention to have a blank like at the end of the file